otwcode/otwarchive
# AO3-6009 Policy & Abuse link in first login banner

# Pull Request Checklist

  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6009

## Purpose

Adds a text and link to the first login message telling users about policy and abuse and what they can do if they experience harassment.  All user-facing strings have been added to localization config to support future translations.

## Testing Instructions

Feature file for the login banner updated to test this.

For manual testing:
- Create new account or re-enable the first login message
- View the first login banner
- Verify the text and link are displayed correctly
- Verity the link takes user to policy and abuse page

## References

Same reference linked in JIRA:
https://citizensandtech.org/2020/06/reducing-harassment-impacts-in-feminism-online/

## Credit

tlee911 (they/them)
